ocaml/xapi/xapi_host.ml

```diff
@@ -2382,3 +2382,52 @@ let set_multipathing ~__context ~host ~value =
   Db.Host.add_to_other_config ~__context ~self:host ~key:"multipathing"
     ~value:(string_of_bool value) ;
   Xapi_host_helpers.Configuration.set_multipathing value
+
```

Comment:
Would it make sense to put the types and conversions into a local module?
```
module SchedGran = struct
  type granularity of Core | CPU | Socket
  let of_string = ..
  let to_string = ..
end

Reply:
Sure, no pb. :)

Reply:
Where do you think I should declare this module ?
In its own file seems a bit too much for me.

Reply:
I just commented on message_forwarding: this type is autogenerated from the datamodel enum, so we can't change it. However, we should put the to/of string functions together (and not define them twice), and the usual place for them is record_util.ml.